src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -353,6 +353,8 @@ protected function instantiateObject(array &$data, string $class, array &$contex
                 $paramName = $constructorParameter->name;
                 $key = $this->nameConverter ? $this->nameConverter->normalize($paramName, $class, $format, $context) : $paramName;
```

Comment:
I guess the name converters should get the attribute context as well

Reply:
There is already a `MetadataAwareNameConverter` which is also getting the metadata out. I'm not sure if we should pass here the metadata context 🤔

Reply:
Hmm 🤔 But it does not fetch the context from metadata as far as I see it, right?

Reply:
I'm not sure but `MetadataAwareNameConverter::getCacheValueForNormalization` is getting the Metdata. But I'm not an expert here so maybe someone else could answer the question?

Reply:
It does only fetch the `@SerializedName`, not the context at all https://github.com/symfony/symfony/blob/289468042b9eac7ef2c2db7e144e4c51ec33a24a/src/Symfony/Component/Serializer/NameConverter/MetadataAwareNameConverter.php#L85

Reply:
Sure, it only handles its own part. So let's pass the context to it from the `AbstractNormalizer`

Reply:
Done with https://github.com/symfony/symfony/pull/46680/commits/1f3d272fafad297b972ed04b7f2774f45c8441ba